controlplane/rosa/controllers/rosacontrolplane_controller.go

```diff
@@ -209,6 +205,44 @@ func (r *ROSAControlPlaneReconciler) reconcileNormal(ctx context.Context, rosaSc
 		rosaScope.ControlPlane.Status.FailureMessage = nil
 	}
 
+	if errs := rosaScope.ControlPlane.Spec.AdditionalTags.Validate(); errs != nil {
+		var msg []string
+		for _, err := range errs {
+			msg = append(msg, err.Error())
+		}
+		rosaScope.ControlPlane.Status.FailureMessage = ptr.To(strings.Join(msg, ", "))
```

Comment:
Hm, I see this is using `FailureMessage`, although Cluster API usually uses this as a terminal state; this is covered in https://cluster-api.sigs.k8s.io/developer/providers/cluster-infrastructure#infracluster-resources

These errors should be either logged, returned, or in a condition

/hold